scripts/gitlab/list_of_specs.json

```diff
@@ -0,0 +1,64 @@
+{ 
+  "toss_3_x86_64_ib":
+  { 
+    "ruby":
+    [
+      "%clang@3.9.1",
+      "%clang@3.9.1+cuda",
+      "%clang@4.0.0",
+      "%clang@4.0.0+cuda",
+      "%clang@6.0.0",
+      "%clang@9.0.0",
+      "%gcc@4.9.3",
+      "%gcc@4.9.3+cuda",
+      "%gcc@6.1.0",
+      "%gcc@6.1.0+cuda",
+      "%gcc@7.1.0",
+      "%gcc@7.1.0+cuda",
+      "%gcc@7.3.0",
+      "%gcc@8.1.0",
+      "%intel@16.0.4",
+      "%intel@17.0.2",
+      "%intel@18.0.0",
+      "%intel@19.0.4",
+      "%pgi@17.10",
+      "%pgi@18.5",
+      "%pgi@19.4"
+    ]
+  },
+  "blueos_3_ppc64le_ib_p9":
+  { 
+    "lassen":
+    [
+      "%clang@3.9.1",
+      "%clang@4.0.0",
+      "%clang@9.0.0",
+      "%clang@9.0.0+libcpp",
+      "%clang@coral2018.08.08",
```

Comment:
These clang coral compilers are pretty out of date.

Reply:
removed this file as well as generate_host_configs.py - a relic that no one uses anymore.